docs/user/install-cli.md

```
@@ -0,0 +1,65 @@
+## Download archive
+
+1. Click the **❔** (question mark) icon in the upper part of the web console page.
+
+2. Choose a downloadable archive using proper architecture and OS. Click the file to download it.
+
+## Installation
+
+1. Install `flightctl`
+   * For **Linux** OS
+     - Decompress the archive file
+       ```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)